packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts

```diff
@@ -55,6 +57,9 @@ export async function pushResources(
   let continueToPush = context.exeInfo && context.exeInfo.inputParams && context.exeInfo.inputParams.yes;
 
   if (!continueToPush) {
+    if (context.exeInfo.iterativeRollback) {
+      context.print.info('The CLI will rollback an the last known iterative deployment.');
```

Comment:
Is "an the" a typo?

Reply:
Ah yes should read `The CLI will rollback the last known iterative deployment.`